### DISCUSSION TITLE
Implement manual vod download via vod ID

### DISCUSSION THREAD

**Discussion by aneeshk95** [MrBrax/LiveStreamDVR#427] [Ideas]
I'm curious to see if it's possible to to implement in the Video Download tab a box in which the user can input a vod id number for vods that are unpublished or not deleted from the twitch servers?

For further clarification, streamers have the ability to unpublish vods from public eye but the .m3u8 link is still accessible through various tools out there. As of today the "fetch vod list" finds the streamer's public vods but I would like to also download unpublished vods and use the various tools in LiveStreamDVR such as export and edit.

Would pay if needed.

**Comment by MrBrax** [comment]
isn't this extremely unreliable, brute forcing cache servers?
